plugin/src/main/java/com/playmonumenta/scriptedquests/commands/TestZone.java

```diff
@@ -0,0 +1,55 @@
+package com.playmonumenta.scriptedquests.commands;
+
+import java.util.LinkedHashMap;
+
+import com.playmonumenta.scriptedquests.Plugin;
+
+import org.bukkit.Location;
+
+import dev.jorel.commandapi.CommandAPICommand;
+import dev.jorel.commandapi.CommandPermission;
+import dev.jorel.commandapi.arguments.Argument;
+import dev.jorel.commandapi.arguments.LocationArgument;
+import dev.jorel.commandapi.arguments.TextArgument;
+
+public class TestZone {
+	public static void register(Plugin plugin) {
+		LinkedHashMap<String, Argument> arguments = new LinkedHashMap<>();
+
+		arguments.put("location", new LocationArgument());
```

Comment:
I believe there's a PRECISE_POSITION variant of this that works better (doesn't require chunks to be loaded)